internal/js/bundle.go

```diff
@@ -217,7 +217,7 @@ func (b *Bundle) populateExports(updateOptions bool, bi *BundleInstance) error {
 				dec.DisallowUnknownFields()
 				if err = dec.Decode(&b.Options); err != nil {
 					if uerr := json.Unmarshal(data, &b.Options); uerr != nil {
-						uerr = beautifyJSONUnmarshalError(data, uerr)
+						uerr = beautifyOptionsJSONUnmarshalError(data, uerr)
```

Comment:
>The original idea of https://github.com/grafana/k6/pull/4602 was to report at least the key with potentially the option in question.

I think this expectation is not really visible from the code, nor testing. I would add a comment explaining it, so I don't have to recall the pull requests, if I hit it for some reason the next time.  

```suggestion
// comment about the intentions here
						uerr = beautifyOptionsJSONUnmarshalError(data, uerr)
```